Attestation logging (#275)
edgelesssys/constellation: docs/docs/getting-started/mini-constellation.md

export KUBECONFIG="$PWD/constellation-admin.conf"
```

> Your cluster now consists of a single control-plane node:

Reviewer: ```suggestion
Your cluster initially consists of a single control-plane node:
```

> ```bash
> $ kubectl get nodes
> NAME              STATUS   ROLES           AGE   VERSION                                                                                                                                                                                                      

Reviewer: ```suggestion
```shell-session
$ kubectl get nodes
NAME              STATUS   ROLES           AGE   VERSION
```

control-plane-0   Ready    control-plane   66s   v1.24.6
```

A worker node will request to join the cluster shortly.

> Before the new worker node is allowed to join the cluster, it's state is verified using remote attestation by the join-service.

Reviewer: ```suggestion
Before the new worker node is allowed to join the cluster, its state is verified using remote attestation by the [join-service](../architecture/components.md#joinservice).
```
